tests/installation/zdup.pm

```diff
@@ -8,6 +8,7 @@ sub run() {
     # precompile regexes
     my $zypper_dup_continue = qr/^Continue\? \[y/m;
     my $zypper_dup_conflict = qr/^Choose from above solutions by number or cancel \[1/m;
+    my $zypper_dup_more_conflict = qr/^Choose from above solutions by number or skip, retry or cancel \[1/m;
```

Comment:
I would not add new regexp, but change the first one to `qr/^Choose from above solutions by number[\s\S,]* \[1/m;`
I'm actually testing this change